javascript-data-model/order-history.js

```diff
@@ -0,0 +1,90 @@
+var orderHistory = [
+  {
+    orderNumber: {
+      beginON: 114,
+      midON: 3941689,
+      endON: 8772232
+    },
+    orderPlaced: new Date(2020, 8, 4),
+    returnCLose: new Date(2020, 9, 7),
+    delivered: new Date(2020, 8, 8),
+    shipTo: 'JS Masher',
+    pictureUrl: null,
+    title: 'JavaScript for impatient programmers',
+    author: {
+      authorLastName: 'Rauschmayer',
+      authorFirstName: 'Dr. Axel'
+    },
+    subtotal: 31.55,
+    total: 34.00
+  },
+  {
+    orderNumber: {
+      beginON: 113,
+      midON: 9984268,
+      endON: 1280257
+    },
+    orderPlaced: new Date(2020, 7, 19),
+    returnCLose: new Date(2020, 8, 19),
+    delivered: new Date(2020, 7, 20),
+    shipTo: 'JS Masher',
+    pictureUrl: null,
+    title: 'The Timeless Way of Building',
+    author: {
+      authorLastName: 'Alexander',
+      authorFirstName: 'Christopher'
+    },
+    subtotal: 41.33,
+    total: 44.53
+  },
+  {
+    orderNumber: {
+      beginON: 114,
+      midON: 2875557,
+      endON: 9059409
+    },
+    orderPlaced: new Date(2020, 7, 4),
+    returnCLose: new Date(2020, 8, 5),
+    delivered: new Date(2020, 7, 7),
+    shipTo: 'JS Masher',
+    pictureUrl: null,
+    title: 'Gamecube Controller Adapter',
+    author: {
+      authorLastName: null,
+      authorFirstName: null
+    },
+    subtotal: 15.98,
+    total: 17.22
+  },
+  {
+    orderNumber: {
+      beginON: 113,
+      midON: 2883177,
+      endON: 2648248
+    },
+    orderPlaced: new Date(2020, 7, 3),
+    returnCLose: new Date(2020, 8, 4),
+    delivered: new Date(2020, 7, 5),
+    shipTo: 'JS Masher',
+    pictureUrl: [
+      null,
+      null
+    ],
+    title: [
+      'GameCube Controller',
+      'The Art of Sql'
+    ],
```

Comment:
Notice how this entry has arrays for values, but the rest of your entries have strings for values. This is not a consistent data model, which will cause issues if you ever tried to use this data. Make sure that your format matches for all orders.